packages/backend/src/accounting/service.ts

```diff
@@ -36,15 +36,15 @@ export interface Deposit {
 }
 
 export interface Withdrawal extends Deposit {
-  timeout?: bigint
+  timeout?: number
 }
 
 export interface TransferOptions {
   sourceAccount: LiquidityAccount
   destinationAccount: LiquidityAccount
   sourceAmount: bigint
   destinationAmount?: bigint
-  timeout: bigint
+  timeout: number
```

Comment:
Tigerbeetle now uses seconds (instead of nanoseconds). Hence, I changed everything to seconds and number. Within the psql accounting, seconds are transformed into milliseconds and bigints. We can discuss whether we should also change that to seconds.

Reply:
From a kind of detached view, I don't think we need to update psql accounting to be seconds. Since we've abstracted the accounting service to work with tigerbeetle or postgres I think we should judge the postgres implementation based on the inherent merits of the time format rather than relative to what tigerbeetle is doing. I can see why we'd be consistent to start but at this point I wouldn't bother personally.

And looking more closely at the code the `timeoutMs` detail does seem to be fully contained to the psql service. Ultimately drives the transfer `state` and `expiresAt` - so its not like there is some environment variable or display field that means one thing if using tigerbeetle and another if using postgres.

Reply:
I made the base time unit seconds because you can easily convert that to milliseconds but you can't go from milliseconds to seconds without rounding.